lib/scripts/create_logs_config.py

```diff
@@ -7,14 +7,19 @@
 import os
 import json
 
-LOGS_CONFIG_DIR = os.environ['LOGS_CONFIG_DIR']
-LOGS_CONFIG = os.environ['LOGS_CONFIG']
-DD_TAGS = os.environ['DD_TAGS']
+LOGS_CONFIG_DIR = os.environ.get('LOGS_CONFIG_DIR')
+LOGS_CONFIG = os.environ.get('LOGS_CONFIG')
+DD_TAGS = os.environ.get('DD_TAGS')
 
```

Comment:
All these variables should always be set, something must have gone wrong if they are not here.

Reply:
`DD_TAGS` is optional, and `LOGS_CONFIG` is only required if the the agent is expected to capture logs.
We can add a log message to mark that it's missing.

Reply:
We only ever run this script when the agent is expected to capture logs, and now we always set `DD_TAGS` in `run-datadog`

Reply:
Yeah, but that could change anytime (we changed the way we export DD_TAGS twice in the last few versions to add support for new features), it's better if this script doesn't depend on it.
So let's just handle the case where it's missing by logging an error message, right?

Reply:
Yes, sounds good!